src/main/webapp/i18n/de/quizExercise.json

```
@@ -261,6 +262,12 @@
                 "individual": "Einzeln",
                 "explanation": "Ausführungsmodus bestimmt wie Studierende an einem Quiz teilnehmen können.\n\nSynchronisiert: Alle Studierenden nehmen gleichzeitig an dem Quiz teil.\nGruppiert: Es gibt mehrere Gruppen an denen Studierende mit einem Passwort teilnehmen können.\nEinzeln: Studierende können jederzeit selbstständig an dem Quiz teilnehmen."
             },
+            "releaseDateExplanation": {
+                "synchronized": "Synchronisiert: Das Datum, an dem die Übung für die Schüler sichtbar wird. Die Schüler können die Fragen erst ab dem Startdatum sehen.",
```

```suggestion
                "synchronized": "Synchronisiert: Das Datum, ab dem das Quiz für die Studierenden in der Aufgabenübersicht sichtbar wird. Die Studierenden können die Fragen erst ab dem Startdatum sehen.",
```
The distinction between releaseDate and startDate is not clear. I added the `in der Aufgabenübersicht sichtbar wird``